server/index.js

```diff
@@ -174,6 +175,8 @@ app.put("/revisequestion/:id", async (req, res) => {
       question,
       correct_answer,
       incorrect_answers,
+      //id_users
+      //id_quiz
```

Comment:
Do we need these?

Reply:
Alex and I are leaving these as reminders that they are available <- We had a difficult time back tracking our way through the front end, and dont want to do that again. I will change the comment.